sklearn/linear_model/_logistic.py

```diff
@@ -755,13 +755,9 @@ def _logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
         .. versionchanged:: 0.22
             Default changed from 'ovr' to 'auto' in 0.22.
 
-    random_state : int, RandomState instance or None, optional, default None
-        The seed of the pseudo random number generator to use when shuffling
-        the data.  If int, random_state is the seed used by the random number
-        generator; If RandomState instance, random_state is the random number
-        generator; If None, the random number generator is the RandomState
-        instance used by `np.random`. Used when ``solver`` == 'sag' or
-        'liblinear'.
+    random_state : int, RandomState instance or None, optional (default=None)
```

Comment:
Following our new style:

```suggestion
    random_state : int, RandomState instance or None, default=None
```

Reply:
None of the other parameters in the file follow this style though

Reply:
True, but from what I can tell there is no style consistency in the file, so we might just go for the correct one here